Doppler.ImageAnalyzer.Api/Features/Analysis/Responses/AnalysisResultResponse.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Doppler.ImageAnalyzer.Api.Features.Analysis.Responses
+{
+    public class AnalysisResultResponse
+    {
+        public List<ImageAnalysisResponse>? AnalysisResult { get; set; }
+        public string? AnalysisResultId { get; set; }
```

Comment:
are needed null values?

Reply:
Yes. When these are defined as non-nullable the compiler show the following:
![image](https://github.com/FromDoppler/doppler-image-analyzer/assets/1985175/29ef18b0-7786-4cfe-85b6-3e1220bc6cc3), and suggested me [some changes like this](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/nullable-warnings?f1url=%3FappId%3Droslyn%26k%3Dk(CS8618)#nonnullable-reference-not-initialized).